bert_e/server/status.py

```diff
@@ -48,16 +48,44 @@ def display():
         for pr_id, queued_commits in queue_data.items():
             if int(pr_id) in [i['id'] for i in merged_prs]:
                 continue
-            line = {'pr_id': pr_id}
+            line = {'pr_id': pr_id, 'hotfix': False}
             for version, sha1 in queued_commits:
                 status = BUILD_STATUS_CACHE[build_key].get(sha1, None)
                 state = status.state if status else 'NOTSTARTED'
                 line[version] = {
                     'sha1': sha1,
                     'status': state,
                 }
+            if len(queued_commits) == 1 and version.count(".") == 3:
```

Comment:
pr can have multiple commits but I feel like it doesn't mean what I think it means

Reply:
ah forgot about this comment in our call @bertranddemiddelaer

Reply:
this double condition will isolate a PR targeting a hotfix branch